script/run_build

```diff
@@ -1,7 +1,4 @@
 #!/bin/bash
-# This file was generated on 2020-12-27T14:33:57+00:00 from the rspec-dev repo.
```

Comment:
Why this header is removed? 🤔

Reply:
Good catch!
I didn't want to close those PRs and open them again, manually copied over this file. Let me quickly restore those lines.

Reply:
The script can force push without reopening prs